### PR TITLE
Fix https://github.com/NoHomey/material-ui-number-input/issues/12

### PR DESCRIPTION
**Pull request by dav-ell**
#12 
I doubt the change to webpack.config.js is needed, but I couldn't get it to compile otherwise.

### CODE REVIEW

**Comment by NoHomey** [comment]
One of the things I notice in 2.2 release of TypeScript was the [Object type](https://github.com/Microsoft/TypeScript/wiki/What%27s-new-in-TypeScript#object-type) but did not had time to update. I think it would be better to use the new builtin type instead of the object literal :smile:  Can you update deps and change it?

File: Dev/demo.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default class Demo extends React.Component<{}, DemoState> {
         };
         this.onError = (error: NumberInputError): void => {
             let errorText: string = '';
-            switch(error) {
+            switch (error) {
```
Comment (dav-ell) [inline Dev/demo.tsx]: Whoops, I must've accidentally pressed the formatter.

Comment (dav-ell) [reply Dev/demo.tsx]: How important are those spaces to you? Haha

Comment (NoHomey) [reply Dev/demo.tsx]: at almost no importance

**Comment by dav-ell** [comment]
The only changes should now be exactly those object literals to the object type. I'm trying to get this package into my project as soon as I can, so a merge would be most appreciated.

Let me know if you need anything else on this PR!

**Pull request merged**